decidim-assemblies/app/commands/decidim/assemblies/admin/update_assembly.rb

```diff
@@ -74,7 +74,28 @@ def attributes
             participatory_scope: form.participatory_scope,
             participatory_structure: form.participatory_structure,
             meta_scope: form.meta_scope,
-            show_statistics: form.show_statistics
+            show_statistics: form.show_statistics,
+            purpose_of_action: form.purpose_of_action,
+            type_of_assembly: form.type_of_assembly,
+            type_of_assembly_other: form.type_of_assembly_other,
+            date_created: form.date_created,
+            created_by: form.created_by,
+            created_by_other: form.created_by_other,
+            duration: form.duration,
+            date_of_inclusion: form.date_of_inclusion,
+            has_closed: form.has_closed,
+            closing_date: form.closing_date,
```

Comment:
Isn't it enough to have a `closed_at` date column? (instead of has_closed & closing_date)

Reply:
`closed_at` seems to refer to when the action happened (created at, published at, etc). `closing_date` seems to be referring to somer date in the future, eg the assembly is temporal and it will be closed in a month

Reply:
also there is a field called `duration`

Reply:
So aren't those fields redundant?

Reply:
They are the fields that product says in the description of the issue... @decidim/product can you clarify this, please?

Reply:
Hi, sorry for answering so late. Actually, only the` closing_date` field is defined in specs, `closed_at` does not appear as a field.
The `duration` field is for differentiating whether an Assembly has an unlimited duration or is limited in time. To show that difference, we established that the field type would be date, so that if an Assembly did not have a date associated with duration, it would be shown as unlimited.
I understand that the redundant field here is `closed_at`.